Tropos/Controllers/TRAppDelegate.m

```diff
@@ -54,4 +89,12 @@ - (BOOL)isCurrentlyTesting
     return [[NSUserDefaults standardUserDefaults] boolForKey:@"TRTesting"];
 }
 
+- (TRCourierClient *)courier;
+{
+    if (!_courier) {
+        _courier = [[TRCourierClient alloc] initWithApiToken: TRCourierAPIToken];
```

Comment:
I think that the application controller should probably hold onto this instance, as that's where the other API integrations live.